### ISSUE TITLE
Assigning name to support level user generates warning

### ISSUE THREAD

**Issue by dasistwas** [Wunderbyte-GmbH/moodle-local_edusupport#31]
Reproduce: When a support user is defined and I then set a support level in the appropriate text field I get that:
```
Event property objectid must be set when objecttable is defined

    line 789 of /lib/classes/event/base.php: call to debugging()
    line 808 of /lib/classes/event/base.php: call to core\event\base->validate_before_trigger()
    line 106 of /local/edusupport/choosesupporters.php: call to core\event\base->trigger()
```

**Comment by dasistwas** [comment]
@winnidat when you have fixed and tested, you can use a commit message with keyword "closes" plus the issue number: "closes #31" in this case: https://github.blog/2013-01-22-closing-issues-via-commit-messages/

**Comment by dasistwas** [comment]
Fixed in d492463d7fae1c12f30ad70735f4d0e8f3a5e482
